apps/teams-test-app/src/pages/TestApp.tsx

```diff
@@ -69,6 +70,7 @@ export const TestApp: React.FC = () => {
   return (
     <>
       <div className="App-container">
+        <ExternalAppCardActionsForCECAPIs />
```

Comment:
Seems like it is here for testing. Leaving a comment so it can be alphabetized later :)

Reply:
Yes... That was for testing purpose.. Thanks for catching it!